tests/visualization_tests/matplotlib_tests/test_optimization_history.py

```diff
@@ -169,3 +185,24 @@ def fail_objective(_: Trial) -> float:
 
     figure = plot_optimization_history(studies, error_bar=True)
     assert len(figure.get_lines()) == 0
+
+
+@pytest.mark.parametrize("direction", ["minimize", "maximize"])
+def test_error_bar_in_optimization_history(direction: str) -> None:
+    def objective(trial: Trial) -> float:
+        return trial.suggest_float("x", 0, 1)
+
+    # Test for error_bar.
```

Comment:
I think we don't have to comment this since it is obvious from the name of the test.
```suggestion
```